fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanTransactionRelationRepository.java

```diff
@@ -25,7 +25,6 @@
 public interface LoanTransactionRelationRepository
         extends JpaRepository<LoanTransactionRelation, Long>, JpaSpecificationExecutor<LoanTransactionRelation> {
 
-    List<LoanTransactionRelation> findByFromTransactionAndRelationType(LoanTransaction loanTransaction,
-            LoanTransactionRelationTypeEnum relationType);
+    List<LoanTransactionRelation> findByFromTransaction(LoanTransaction loanTransaction);
```

Comment:
Let's name the parameter as fromTransaction just for consistency with the method naming.

Reply:
good idea